### DISCUSSION TITLE
Mobile apps

### DISCUSSION THREAD

**Discussion by NatureMuncher** [ampache/ampache#3026] [Ideas]
Is there a mobile app for Android? Might be a nice way to get some revenue flowing. I'd buy it in a minute.

**Comment by kuzi-moto** [comment]
There's no "official" app at least. Ideally I think that the front-end would get a facelift and become a PWA, so that a separate app isn't required.

I think everyone that works on Ampache does it out of the love for the software. Accepting money would make it into more of a job since there would at least be some sort of obligation to make things work.

**Reply by lachlan-00** [reply]
Personally, (while I'm running things) all official project software will be free to use and free of cost.

I have no problem with API clients or people making money off the Ampache server itself but i want it to always be completely free and unrestricted to anyone.

We do take donations to pay for the demo sites and subs required for that but it's not important as the costs are low and I will just pay the difference when the time comes for renewal.

If we ever have an official IOS/Android project it would only be allowed to ask for donations and not restrict function. (The developer of the app could ask for donations in app though.)

For the moment though i recommend DSub on android (which i paid for years ago)

Thanks for your support though as we really do it for the love :)
